core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java

```diff
@@ -112,7 +112,7 @@ public LocalFileLock(String lockFileLocation, String lockFile) {
         status = true;
       }
     } catch (IOException e) {
-      status = false;
+      // status = false;
```

Comment:
Should it print error message for IOException ?

Reply:
it is no need

Reply:
I suggest we print a warning message, and remove this assignment
@ravipesala @QiangCai

Reply:
Yes, I think HdfsFileLock has similar problem. Actually there is a PR #2878 trying to fix all exception code style problem, by @kevinjmh . I suggest we can have it handled in that PR. Then for this PR, I think you can remove the assignment directly, same for HdfsFileLock

Reply:
> Yes, I think HdfsFileLock has similar problem. Actually there is a PR #2878 trying to fix all exception code style problem, by @kevinjmh . I suggest we can have it handled in that PR. Then for this PR, I think you can remove the assignment directly, same for HdfsFileLock

ok, I see